### PR TITLE
Use shared summarystate

### PR DESCRIPTION
**Pull request by joakim-hove**
When this is ready and merged the simulator side of UDQ and ACTIONX is complete&trade;

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
> the simulator side of UDQ and ACTIONX is complete™

ehhh - close at least ... :-)

**Comment by joakim-hove** [comment]
I'll merge this later today.

**Comment by joakim-hove** [comment]
Ahh - sorry; that merge was a bit before I intended - pressed Green in wrong project; but I was intending to merge it anyway - I'll leave it.

File: opm/core/wells/WellsManager.cpp [hunk 1]
```diff
@@ -200,6 +200,7 @@ namespace Opm
     /// Construct wells from deck.
     WellsManager::WellsManager(const Opm::EclipseState& eclipseState,
                                const Opm::Schedule& schedule,
+                               const SummaryState& summaryState,
                                const size_t timeStep,
```
Comment (totto82) [inline opm/core/wells/WellsManager.cpp]: What is your plan with adding summarystate to the WellsManager?

Comment (joakim-hove) [reply opm/core/wells/WellsManager.cpp]: image

Comment (joakim-hove) [reply opm/core/wells/WellsManager.cpp]: Hehe - seriously; the `SummaryState` is required context to evaluate dynamic (i.e. UDA based) controls.

**Pull request merged**